### ISSUE TITLE
focusTel issues after #23

### ISSUE THREAD

**Issue by EastEriq** [PolishookDavid/LAST_OCS#36]
After https://github.com/PolishookDavid/LAST_OCS/commit/093f6322450b5262843cfc1e4d63e08fb0d9adae, about #23:
1. `focusTelInSlave` generates plots without fitting parabolas. Original feature or new issue? AFAIR parabolas used not to be drawn, or drawn multiple times, in corner cases with outliers, but it doesn't seem the case here
![immagine](https://github.com/user-attachments/assets/b35bf4a0-ee5e-46eb-821f-ad43d3a71c3d)
2. it is reported that FWHM is no more updated after the focusLoop has been run, and the values shown are the last ones of the loop, even after having observed several targets with `obsByPriority2`
![immagine](https://github.com/user-attachments/assets/63a8ee38-38d6-4e4b-b938-b071f7058f31)
Here the new code was in effect only on Unit 10.

**Comment by EastEriq** [comment]
`S.queryCallback("Unit.Slave(1).Responder.query('[Unit.Camera](http://unit.camera/){1}.ComputeFWHM')",10)` reports `'omit'` instead of the expected default `'last'`

**Comment by PolishookDavid** [comment]
Please also add units (arcsec, focus position) to the axes labels

**Comment by EastEriq** [comment]
For the run of yesterday, to which the images refer, pngs were also not created in `~/log/focus_plots`. I suspect that `focusTelInSlave` did not run to the end because of some error, and didn't restore `.ComputeFWHM`. I have only the master log which says:
```
Sent mount to focus coordinates
Focusing telescope 1 FAILED!
Focusing telescope 2 FAILED!
Focusing telescope 3 FAILED!
Focusing telescope 4 FAILED!
{17:13:48.750|obs.unitCS[10]} Checking definitions and connections of unit 10:
{17:13:58.771|obs.util.Messenger[10_slave_1.Messenger]} 10_slave_1.Messenger timed out waiting for a reply to "true"
{17:13:59.038|obs.unitCS[10]} Slave 1 status: "notresponding"
{17:14:09.053|obs.util.Messenger[10_slave_2.Messenger]} 10_slave_2.Messenger timed out waiting for a reply to "true"
{17:14:09.335|obs.unitCS[10]} Slave 2 status: "notresponding"
{17:14:19.357|obs.util.Messenger[10_slave_3.Messenger]} 10_slave_3.Messenger timed out waiting for a reply to "true"
{17:14:19.621|obs.unitCS[10]} Slave 3 status: "notresponding"
{17:14:29.639|obs.util.Messenger[10_slave_4.Messenger]} 10_slave_4.Messenger timed out waiting for a reply to "true"
{17:14:29.902|obs.unitCS[10]} Slave 4 status: "notresponding"
{17:14:29.903|obs.unitCS[10]} check failed!
{17:15:11.899|obs.util.Messenger[10_slave_3.Messenger]} 10_slave_3.Messenger receiver reports: The logical indices in position 1 contain a true value outside of the array bounds.
{17:15:18.475|obs.util.Messenger[10_slave_1.Messenger]} 10_slave_1.Messenger receiver reports: The logical indices in position 1 contain a true value outside of the array bounds.
{17:15:30.953|obs.util.Messenger[10_slave_4.Messenger]} 10_slave_4.Messenger receiver reports: The logical indices in position 1 contain a true value outside of the array bounds.
{17:15:42.515|obs.util.Messenger[10_slave_2.Messenger]} 10_slave_2.Messenger receiver reports: The logical indices in position 1 contain a true value outside of the array bounds.
```

**Comment by EastEriq** [comment]
I thought that Slaves were spawned by default with a listener MasterMessenger, but they weren't. `.focusTel` was therefore not blocking, causing `.operateUnit` premature continuation commands while `.focusTelInSlave` was running, and mixup. Formerly `operateUnit` was using `.checkFocusTelSuccess` which was sitting there, but removed it. Let's see if the corrections are enough.

**Comment by EastEriq** [comment]
What would remain to do here are embellishments to the plots, either those generated on the fly by `.focusTelInSlave`, which I want to suppress (because the X channel prevents disowning, see https://github.com/EastEriq/LAST_Messaging/issues/4), or those generated a posteriori by `obs.util.observation.plotFocusData`. The former include parabolic fits without or with outliers depending on the case, with arbitrary threshold parameters, whose concept may change if the focus algorithm is changed, as it is badly called for. I think therefore that at this stage is sufficient only to highlight the  best value estimated, without graphical hints of how.
